include/libdnf/base/transaction.hpp

```diff
@@ -80,6 +80,10 @@ class Transaction {
     /// @return the transaction groups.
     std::vector<libdnf::base::TransactionGroup> & get_transaction_groups() const;
 
+    /// Check the transaction by running it with RPMTRANS_FLAG_TEST.
+    /// @return An enum describing the result of the transaction
```

Comment:
I have one suggestion - <please could you mention that the test is automatically performed by run() therefore calling test() and run() will be not efficient and not needed.

Reply:
Done, thanks.